src/App.tsx

```diff
@@ -62,6 +62,16 @@ const App: FC = () => {
         })
       }
     }
+    if (CLOUD && isFlagEnabled('coveo')) {
+      const script = document.createElement('script')
+      script.src =
+        'https://platform.cloud.coveo.com/rest/organizations/influxdatanonproduction1eh1cn7go/pages/95a20052-218d-4048-9081-16dbcebbdebb/inappwidget/loader'
```

Comment:
is it safe to have this api-key embedded url in this open source repo?

Reply:
Yes, this is a public key that the Coveo team has sent us, they can invalidate it once we are done demonstrating this feature, and its not going to be connected to billing or anything that would endanger our users, at least not in any way im aware of.

Reply:
More worried about it endangering us - if someone decided they wanted to try out Coveo in their app and decided to use our url 😆